parallel-crypto3/libs/parallelization-utils/test/thread_pool.cpp

```diff
@@ -55,7 +55,8 @@ BOOST_AUTO_TEST_CASE(vector_multiplication_test) {
         }, nil::crypto3::ThreadPool::PoolLevel::HIGH));
 
     for (std::size_t i = 0; i < size; ++i) {
-        BOOST_CHECK(v[i] == i * i);
+        if(v[i] != i * i) // If uncomment, test result parser will log an error: CData section too big found
```

Comment:
looks strange.
Condition should be checked in BOOST_CHECK

Reply:
It cause error on the parser side:
https://github.com/NilFoundation/placeholder/actions/runs/12015936613/job/33495224928

Reply:
`BOOST_CHECK_EQUAL` helps?

Reply:
trying now

Reply:
@akokoshn  that didn't help
https://github.com/NilFoundation/placeholder/actions/runs/12031864193/job/33545980584

Reply:
I fixed it a different way